### DISCUSSION TITLE
Component style with Device Manager

### DISCUSSION THREAD

**Discussion by rbuyse** [GrapesJS/grapesjs#5213] [Q&A]
Hi,

Thank you very much for GrapesJs, it's a great tool!

I'm having an issue since I've used Device Manager. For example, on the Mobile vue, when I add a component that have a "style" option, it now adds a media query on my css and the block doesn't appear well on desktop.

Is is possible to disable this option somewhere?

Thanks you again and have a great day

<img width="222" alt="Capture d’écran 2023-07-04 à 12 18 57" src="https://github.com/GrapesJS/grapesjs/assets/132270741/52545608-2fab-42c7-8045-ac2c6d5c61df">

<img width="222" alt="Capture d’écran 2023-07-04 à 12 18 57" src="https://github.com/GrapesJS/grapesjs/assets/132270741/9318aba4-335d-4472-afc8-ea4ee2e7d39a">

**Comment by bgrand-ch** [comment]
@artf Same problem, do you have a solution please? 🤔

**Comment by artf** [comment]
Seems to be an old behaviour, but honestly not sure how this would ever might be useful assigning those component defined styles to current selected device... I'll fix it in next release

**Reply by bgrand-ch** [reply]
Thanks a lot 🙏
